ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -35,7 +39,21 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.17
   to: hypershift
+- dockerfile_literal: |-
```

Comment:
@alvaroaleman I'm surprised by this, I didn't think it was necessary. Can't we use `oc` in a step out of the box somehow? Something like:

```
steps:
  - as: execute
    cli: latest
```

Reply:
To get `oc`, yes. To get `expect`, no

Reply:
I don't see `expect` being used though so maybe that is just here because of some copy paste?

Reply:
Didn't try this approach, but this one works in other places. I will try your suggestion.